content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -1,92 +1,106 @@
+   - Tables 2, 4, and 5: These tables are available.
+   - Tables 2, 4, and 5: These tables' total capacity is, respectively, 2, 4, and 8 people.
+   - Table 1: The table of two is full.
+   - Table 3: The table of four is taken by one person.
+   - Table 5: "Famous Joe" has a reservation for 8 people at 12:00.
 
-Once your *Point of Sale* has been configured, select *Table
-Management* under :menuselection:`Point of Sale --> Configuration -->
-Point of Sale`. Then, click on *Floors* to create and name your floor and tables.
+Configuration
+=============
 
-.. image:: floors_tables/restaurant_03.png
-    :align: center
-    :alt: View of the table management feature. Way to manage and create floors for a pos
+From the POS backend
+--------------------
 
-.. image:: floors_tables/restaurant_04.png
-    :align: center
-    :alt: Backend view of a restaurant floor. Table name and number of sits for each table
+To create floors and tables from the backend, go to :menuselection:`Configuration --> Floor Plans`,
+and click :guilabel:`New` to create a floor. Name it, select the related point(s) of sales, and
+click :guilabel:`Add a line` to create a table. Name the table and assign a number of seats. You can
+also link it to an appointment resource to make the table bookable. Once done, click :guilabel:`Save
+& Close` or :guilabel:`Save & New` to confirm.
+
+.. image:: floors_tables/table-creation-backend.png
+   :scale: 75%
+   :alt: window to create a table in the POS backend
 
 .. note::
-   Don’t forget to link your floor to your point of sale.
+   The POS must be opened and :ref:`edited from the frontend <floors_tables/frontend>` to create a
+   map of your restaurant or bar reflecting your actual floor plan.
 
-Add tables
-==========
+.. tip::
+   Create floors on the spot: :ref:`access your POS settings <configuration/settings>` and type your
+   floor name into the :guilabel:`Floor` field of the :guilabel:`Floors & Tables Map` category and
+   press *enter*.
 
-To add tables, you can also open your PoS interface to see your
-floor(s).
+   .. image:: floors_tables/floor-creation-backend.png
+      :scale: 75%
+      :alt: setting to create floors from the POS settings
 
-.. image:: floors_tables/restaurant_05.png
-    :align: center
-    :alt: View of the floors menu to manage several floors at the same time
+.. _floors_tables/frontend:
 
-Then, click on *Edit Mode* (pencil icon on the upper right corner) to be allowed to
-create, move, modify tables, etc.
+From the POS frontend
+---------------------
 
-.. image:: floors_tables/restaurant_06.png
-    :align: center
-    :alt: View of the floor management. Add tables, the number of sits, their name and their shape
+To create floors and tables from the frontend, :ref:`open a POS session <pos/session-start>`, click
+the hamburger menu icon :guilabel:`≡` in the upper right corner, and :guilabel:`Edit Plan` to enter
```

Comment:
```suggestion
the hamburger menu icon :guilabel:`≡` in the upper right corner, then :guilabel:`Edit Plan` to enter
```
since you click on both? probably nitpicking though ^^